### PR TITLE
CMS-719: README file

### PR DESCRIPTION
**Pull request by rwagner00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Wordpress Bedrock Integrations
+
+[![Unsupported](https://img.shields.io/badge/Pantheon-Unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
+
+## About
+This repo contains settings, configuration, and functionality to enable a WordPress site to work on Pantheon.
```
Comment (jspellman814) [inline README.md]: We should clarify the upstream this is intended to be used with (`wordpress-bedrock-recommended`), and that it is [included in the upstream](https://github.com/pantheon-systems/wordpress-bedrock-recommended/blob/master/composer.json#L25)  by default. See: https://github.com/pantheon-systems/wordpress-bedrock-integrations/blob/main/composer.json#L3

And maybe we call out how this repo is being used within `wordpress-bedrock-recommended` (e.g. This repo is pulled into Pantheon's WordPress Bedrock Recommended upstream, and its contents are placed into the necessary places via the [cleanup-composer script](https://github.com/pantheon-systems/wordpress-bedrock-recommended/blob/master/scripts/composer/cleanup-composer#L13)). We should also document the cleanup-composer script in the upstream, but adding a note about it here should give folks some context as to what purpose this repo serves.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Wordpress Bedrock Integrations
+
+[![Unsupported](https://img.shields.io/badge/Pantheon-Unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
+
+## About
+This repo contains settings, configuration, and functionality to enable a WordPress site to work on Pantheon. 
+
+## Installation
+Add this to a project via composer: `composer require pantheon-systems/wordpress-bedrock-integrations`.
```
Comment (jspellman814) [inline README.md]: Not certain about including this line, as installing this repo as is doesn't actually do anything without additional scripting.

**Review by jspellman814 — CHANGES_REQUESTED** [review]

**Comment by rwagner00** [comment]
@jspellman814 Updated!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Wordpress Bedrock Integrations
+
+[![Unsupported](https://img.shields.io/badge/Pantheon-Unsupported-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#unsupported)
+
+## About
+This repo contains settings, configuration, and functionality to enable a WordPress site to work on Pantheon. It is included by default in the Pantheon standard WordPress upstream, and its contents are placed into their correct locations via the cleanup-composer script.
```
Comment (jspellman814) [inline README.md]:
```suggestion
This repo contains settings, configuration, and functionality to enable a WordPress site to work on Pantheon. It is included by default in Pantheon's [Composer-enabled WordPress upstream](https://github.com/pantheon-systems/wordpress-bedrock-recommended), and its contents are placed into their correct locations via the [cleanup-composer script](https://github.com/pantheon-systems/wordpress-bedrock-recommended/blob/master/scripts/composer/cleanup-composer). 
```

**Comment by jspellman814** [comment]
One small change, but otherwise looks good to me!

**Pull request merged**